packages/components/src/components/content-expander/content-expander.jsx

```diff
@@ -20,27 +20,24 @@ const ContentExpander = ({
     onToggle,
     measure,
 }) => {
-    const [is_visible, toggleVisibility] = React.useState(is_expanded);
```

Comment:
Removing this will break the storybook.

Reply:
@zaki-binary  Oh okay, that might break some things. Actually it's better with the state, but I had to remove it because for some reason when the `is_expanded` prop changes, the stat doesn't change. It only changes if you on click

Reply:
Okay nevermind, I found a fix 👍